react/features/base/session/reducer.js

```diff
@@ -0,0 +1,27 @@
+import { ReducerRegistry } from '../redux';
+
+import { SESSION_CREATED, SESSION_FAILED, SESSION_TERMINATED } from './actionTypes';
+
+ReducerRegistry.register('features/base/session', (state = new Map(), action) => {
+    switch (action.type) {
+    case SESSION_CREATED: {
+        const { session } = action;
+        const nextMap = new Map(state);
```

Comment:
Since we onlhy allow one session at a time I wonder if having this map is a good idea. We could have an array of sessions and maybe a `currentSession` if we feel the need?

Reply:
and what would better about using an array here ?

Reply:
Maybe lets keep it open and will get back to it later once we get other features to use the base/session

Reply:
👍